shell/src/main/java/alluxio/cli/fs/command/LsPathInAlluxio.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.cli.fs.command;
+
+import alluxio.AlluxioURI;
+import alluxio.client.file.FileSystemContext;
+import alluxio.client.file.URIStatus;
+import alluxio.exception.AlluxioException;
+import alluxio.grpc.ListStatusPOptions;
+import alluxio.util.FormatUtils;
+import alluxio.wire.BlockLocation;
+import alluxio.wire.FileBlockInfo;
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.Options;
+
+import javax.annotation.Nullable;
+import java.io.IOException;
+import java.util.*;
+
+public class LsPathInAlluxio extends AbstractFileSystemCommand {
+
+    private static final String READABLE_OPTION_NAME = "h";
+
+    private static final Option READABLE_OPTION =
+            Option.builder(READABLE_OPTION_NAME)
+                    .required(false)
+                    .hasArg(false)
+                    .desc("print sizes in human readable format (e.g., 1KB 234MB 2GB)")
+                    .build();
+
+    public LsPathInAlluxio(@Nullable FileSystemContext fsContext) {
+        super(fsContext);
+    }
+
+    /**
+     * Displays information for In-Alluxio data size under the path group by worker.
+     *
+     * @param path The {@link AlluxioURI} path as the input of the command
+     * @param readable whether to print info of human readable format
+     */
+    private void lsPathInAlluxio(AlluxioURI path, boolean readable) throws AlluxioException, IOException {
+
+        Timer timer = new Timer();
+        timer.schedule(new TimerTask() {
+            @Override
+            public void run() {
+                System.out.println("Getting directory status of %s files or sub-directories "
```

Comment:
If you want to give a parameter to the `%s`, you will need to use `System.out.format("%s%n", x);` where `%n` is the newline character.